dotCMS/src/main/java/com/dotcms/experiments/business/ExperimentsAPIImpl.java

```diff
@@ -527,10 +527,76 @@ public Experiment start(String experimentId, User user)
         return toReturn;
     }
 
+    @Override
+    public Experiment forceStart(String experimentId, User user)
```

Comment:
any test for this method?

Reply:
I think we need a better doc on this method, for example we need to explain what if exists any RUNNING Experiment and the new one is on SCHEDULED status so the old one is going to be END just if there is any conflicts on the Start/End Dates

Reply:
It's documented already at API level, not Impl class. Please check it out @freddyucv